kernelci/kbuild.py

```diff
@@ -533,6 +553,15 @@ def _build_kselftest(self):
         self.addcmd("make -j$(nproc) kselftest-gen_tar" + " 1> " +
                     self._af_dir + "/build_kselftest.log" +
                     " 2> " + self._af_dir + "/build_kselftest_stderr.log")
+
+    def _build_dtbs_check(self):
+        """ Check if dtbs are present """
+        self.startjob("build_dtbs_check")
+        self.addcmd("cd " + self._srcdir)
+        self.addcmd("make -j$(nproc) dtbs_check" + " 1> " +
```

Comment:
This needs a `.config` to be present. IMHO this should be the base defconfig for the target architecture, as defined in the `build_variants` section of [pipeline.yaml](https://github.com/kernelci/kernelci-pipeline/blob/main/config/pipeline.yaml#L444), e.g. `multi_v7_defconfig` for armv7.

Reply:
You are right, i adjusted code a bit